Extension/src/LanguageServer/client.ts

```diff
@@ -792,6 +801,7 @@ export interface Client {
     addFileAssociations(fileAssociations: string, languageId: string): void;
     sendDidChangeSettings(): void;
     isInitialized(): boolean;
+    ShowConfigureIntelliSenseStatus(): boolean;
```

Comment:
`showConfigureIntelliSenseStatus` (little 's')

Reply:
It was capitalized to avoid a naming conflict with the lowercased variable name. I saw other methods start with a capital up above. I could change it though still.

Reply:
Yeah, if you have a suggestion for what to rename showConfigureIntelliSenseStatus to.

Reply:
You can leave it for now then.  Or call it `showConfigureIntelliSenseButton`.

Reply:
Sure, ending with Button is fine with me, although it makes it longer. But the issue with Capitalize and Lowercase still exists unless we can think of some better naming.

Reply:
FWIW, replace "Status" with "Button" is the same number of characters.

Reply:
Oh -- I added Button onto Status -- that's not what you wanted, i.e. StatusButton?

Reply:
I don't really care that much.  I thought it would be easier to just call it the ConfigureIntelliSenseButton instead of adding extra adjectives to it.  It's not really a "status".  It goes on the "status bar", but I didn't want all of that in the name either.

Reply:
Sure, I went with the shorter Button.